src/porepy/numerics/vem/dual_elliptic.py

```diff
@@ -289,7 +289,7 @@ def _velocity_dof(self, g, g_m, grid_swap):
         return hat_E_int
 
     def assemble_int_bound_flux(
-        self, g, data, data_edge, grid_swap, cc, matrix, self_ind
+        self, g, data, data_edge, grid_swap, cc, matrix, self_ind, coefficient = 1.
     ):
```

Comment:
1) This breaks the uniform interface of all EllipticDiscretizations. I don't like it at all, with a unit default or not. What is the case where you need to do this? If it is a special case dedicated for a paper, and I will quite likely characterize it as such, the preferred solution is to write a dedicated subclass with the desired behavior.
2) If we are to do this, you will need to allow it for fv methods as well. And documentation of what actually happens, and which equation is solved must be in place. But don't bother until you have convinced me that this should actually be included.
3) You need to run black.

Reply:
OK, I see it will be difficult to convince you to but this update in the core :P
I would probably write a subclass in the paper folder, if this will be needed for other applications in the future we might move the modifications in the core.

Reply:
Actually it's quite annoying to "just" inherit and make the change.. Cause I need to inherit again to obtain the implementation of the MVEM.. I can propose the change also for the FV solver? I checked it and it looks less cumbersome